sdcflows/interfaces/bspline.py

```diff
@@ -76,6 +75,11 @@ class _BSplineApproxInputSpec(BaseInterfaceInputSpec):
         usedefault=True,
         desc="generate a field, extrapolated outside the brain mask",
     )
+    zooms_min = traits.Float(
+        1.4,
```

Comment:
I'm seeing this issue on 1.6mm and 1.8mm. I might set this as 2.0 by default, or 1.95 to be sure that we don't treat 1.98 differently from 2.0.

Would we consider a 4:1 voxel:knot ratio acceptable in the worst case? With a 16mm knot spacing and using a 2-or-nothing down-sampling ratio as proposed below, a threshold of 1.95 would get us a range of 8:1.95 to 8:1.

Reply:
Sounds good to me. I think 4:1 would be good in the worst case.

Reply:
Changed to accepting tuples (i.e., anisotropic voxels).

I believe that, if we want some ratio, we should calculate the target spacings elsewhere. That looks like a good option for rodents, but I don't think we should worry about that right this minute.